src/helpers/general.helpers.ts

```diff
@@ -0,0 +1,29 @@
+export const fileExists = async (file: string): Promise<boolean> => {
```

Comment:
praise: this is nice; well done.

question: maybe we rename `helpers` to `utils` and remove the `.helpers.ts` extension?